tests/v2/validation/provisioning/rke2/hardened_cluster_test.go

```diff
@@ -5,18 +5,19 @@ package rke2
 import (
 	"testing"
 
+	"github.com/rancher/rancher/tests/v2/actions/charts"
+	"github.com/rancher/rancher/tests/v2/actions/clusters"
+	"github.com/rancher/rancher/tests/v2/actions/projects"
+	"github.com/rancher/rancher/tests/v2/actions/provisioning"
+	"github.com/rancher/rancher/tests/v2/actions/provisioninginput"
+	"github.com/rancher/rancher/tests/v2/actions/reports"
+	"github.com/rancher/rancher/tests/v2/actions/users"
 	cis "github.com/rancher/rancher/tests/v2/validation/provisioning/resources/cisbenchmark"
 	"github.com/rancher/shepherd/clients/rancher"
 	"github.com/rancher/shepherd/clients/rancher/catalog"
 	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
-	"github.com/rancher/shepherd/extensions/charts"
-	"github.com/rancher/shepherd/extensions/clusters"
+	extensionscluster "github.com/rancher/shepherd/extensions/clusters"
```

Comment:
not a fan of having duplicate action/extension package names that we have to fix like this on the test level. Would it be worth just renaming the actions packages so there's not this overlap?

Reply:
Good question. Let me think about this.

Reply:
So Caleb I thought about this more, I think I rather have the test writer write their own package alias to how they see fit, rather than imposing a specific package naming scheme

Reply:
I agree, I think the authors should be able to name their aliases however they see fit

Reply:
I think this will come up on basically every package so it will be a little tedious to do imo. But I understand where you are coming from as well so I can be good either way if others prefer it this way.